### ISSUE TITLE
It looks like the dropdown search for non-ascii characters is not works after the version 0.13.0 of dash-core-components

### ISSUE THREAD

**Issue by lumingdong** [plotly/dash-core-components#118]
It looks like the dropdown search for non-ascii characters is not works after the version 0.13.0 of dash-core-components

**Comment by chriddyp** [comment]
@lumingdong - Could you share a small reproducable example? I tried 

```
# -*- coding: utf-8 -*-

import dash
import dash_html_components as html
import dash_core_components as dcc

print(dcc.__version__)

app = dash.Dash()

app.layout = html.Div([
    dcc.Dropdown(
        options=[
            {'label': 'Montréal', 'value': 'Montréal'},
            {'label': 'Tokyo', 'value': 'Tokyo'},
            {'label': '北京', 'value': '北京'},
        ]
    )
])


if __name__ == '__main__':
    app.run_server(debug=True)
```

and it worked for me. This is the test case that I am using (test code: https://github.com/plotly/dash-core-components/blob/bc06b70fa81445d3cca88d44a9ea6ab510756fe1/test/test_integration.py#L326)

![chinese-characters](https://user-images.githubusercontent.com/1280389/33279774-c7b1cb72-d36d-11e7-8cb8-915144b483ab.gif)

**Comment by lumingdong** [comment]
@chriddyp Sorry, I did't  test the 0.15.2, because there is no such version before, the following is my test results for some version.
code：
`# -*- coding: utf-8 -*-

import dash
import dash_html_components as html
import dash_core_components as dcc

# print(dcc.__version__)

app = dash.Dash()

app.layout = html.Div([
    html.H1('dash_core_components version: {0}'.format(dcc.__version__)),
    dcc.Dropdown(
        options=[
            {'label': 'Montréal', 'value': 'Montréal'},
            {'label': 'Tokyo', 'value': 'Tokyo'},
            {'label': '北京', 'value': '北京'},
        ]
    )
])


if __name__ == '__main__':
    app.run_server(debug=True)`
![0 13 0](https://user-images.githubusercontent.com/25942243/33299867-89a1648a-d428-11e7-9268-216ec2e658e6.gif)
![0 14 0](https://user-images.githubusercontent.com/25942243/33299874-8d7f3c3a-d428-11e7-95de-b8f47c0c4a57.gif)
![0 15 0](https://user-images.githubusercontent.com/25942243/33299878-91ab5da2-d428-11e7-9b2b-6479b30eaff9.gif)
![0 15 2](https://user-images.githubusercontent.com/25942243/33299885-95542024-d428-11e7-8b86-504a0bc69dce.gif)

**Comment by chriddyp** [comment]
Thanks for testing these @lumingdong !! It looks like this got fixed in 0.15.2. Is this issue good to close?

**Comment by lumingdong** [comment]
@chriddyp Yes, It got fixed in 0.15.2，Thanks you.

**Comment by chriddyp** [comment]
Thanks again for reporting @lumingdong :)
